src/installation/guides/zfs.md

```diff
@@ -0,0 +1,225 @@
+# ZFS On Root
+
+This installation guide assumes that an already existing Void Linux install is
+available. Not a Live USB. If an existing Void install does not exist, please
+consult one of the [other installation guides](./index.md) or use a custom iso
+such as [hrmpf](https://github.com/leahneukirchen/hrmpf) before continuing with
+this guide.
+
+## Setup
+
+From the permanent system, install the `zfs` package.
+
+Then ensure the ZFS module is loaded with
+[modprobe(1)](https://man.voidlinux.org/modprobe):
+
+`# modprobe zfs`
+
+Locate the disk to format using [fdisk(8)](https://man.voidlinux.org/fdisk.8) or
+[lsblk(8)](https://man.voidlinux.org/lsblk.8):
+
+`# fdisk -l`
+
+## Partitioning
+
+Prepare the drive for installation by creating one of the following partition
+schemes using [cfdisk(8)](https://man.voidlinux.org/cfdisk.8) or another
+partitioning tool:
+
+`# cfdisk -z /dev/sda`
+
+**Warning:** The disk being partitioned will be formatted and any existing data
+on the disk will be destroyed.
+
+For a BIOS/MBR system:
+
+```
+Partition    Size       Type
+1             ?G     Solaris Root(bf00)
+```
+
+For a BIOS/GPT system:
+
+```
+Partition    Size       Type
+1             2M     BIOS boot    
+2             ?G     Solaris Root
+```
+
+ZFS pools should use id's to maintain universal functionality across computers
+and to prevent the misrecognition of drives.
+
+Id's can be located in the `/dev/disk/by-id` folder:
```

Comment:
All instances of `Id's` should instead be `IDs`, an apostrophe implies possession - we simply want the plural form here. `folder` should instead be `directory`.